xalia/Uia/UiaElement.cs

```diff
@@ -472,11 +475,15 @@ protected override void DeclarationsChanged(Dictionary<string, (GudlDeclaration,
 
             if (ElementWrapper.Hwnd != IntPtr.Zero)
             {
-                int win32_element = -1, win32_listview = -1, win32_tabcontrol = -1, win32_trackbar = -1;
+                int msaa_element = -1, win32_element = -1, win32_listview = -1, win32_tabcontrol = -1, win32_trackbar = -1;
 
                 for (int i = 0; i < Children.Count; i++)
                 {
-                    if (Children[i] is Win32Trackbar)
+                    if (Children[i] is UiaElement)
+                        continue;

```

Comment:
Why is a continue now needed here when it wasn't before? As an early out condition? IIUC, there shouldn't be a case where `is MsaaElement` and `is UiaElement` would both evaluate to true, right? just wanted to make sure I'm not misunderstanding.

Reply:
The continue is not needed, it's an optimization. Most children will be UiaElement, so there's no need to go through all the other type checks.

Reply:
Really, this should be rewritten using the new RegisterTrackedProperties/TrackedPropertyChange methods.

Reply:
DeclarationsChanged should just be retired.

Reply:
Makes sense. It doesn't hurt anything by having it in this patch, but it could be potentially confusing to someone looking at the commit history since adding the continue isn't necessary for MSAA support to work. Just a thought.

Reply:
Pending as #2.